### ISSUE TITLE
Auto battles setup for food not working

### ISSUE THREAD

**Issue by 3bee88** [FarbstoffRSL/RSL-Helper#440]
### Operating system

Windows 10

### Systemlanguage

English (EN)

### Current Game-Version

5.80.0

### RSL Helper Version

v3.4.2.7

### Bug description

Auto battles for food not working and I get this error when it happens. I've done both using the auto select to choose the food champions and just selecting them myself and it doesn't seem to work. Restarted rslhelper and deleted and re-downloaded it multiple times and nothing has changed. Not sure if I'm just doing something wrong since i have not seen anyone have this issue so far.

![image](https://user-images.githubusercontent.com/111711638/185926034-725f6d28-346a-4bb2-a043-41f032c748e0.png)

### Steps to reproduce

Click on multi-level xp do set-up for food and then run auto battles.

**Comment by Dwza** [comment]
silly to ask, but you running the helper as admin?
and is he running some runs or does this appear straight away?

**Comment by 3bee88** [comment]
I only run as admin cause it wouldn't open otherwise. It will do the run and when it attempts to do the second run the error pops up. This only happens when using the multi-level xp option. When i do dragon runs with only the options run resources effectively and sell artifacts checked off it runs fine.

**Comment by Dwza** [comment]
if you turn all other options off (autosell, resources effectively....) and only use multi, does this still appear?

**Comment by 3bee88** [comment]
Just tried with all of the options taken out except of multi and it still appears. 

![image](https://user-images.githubusercontent.com/111711638/186141186-45824172-82f0-4b66-9ab0-2400d93bffbf.png)

**Comment by floydmcboyd** [comment]
I'm having the same issue. The different settings don't seem to matter. It completes the first run and then the modulus error shows and it never starts a second run.

**Comment by kojikeneda** [comment]
I'm having the same issue. The different settings don't seem to matter. It completes the first run and then the modul error shows and it never starts a second run.

**Comment by ma-KaBre** [comment]
Worked for me on three different accounts today.

**Comment by 3bee88** [comment]
Well deleted rslhelper along with the folder that has the data and re-downloaded the files again just to see if that would change anything and nothing has changed as shown in photo below:

![image](https://user-images.githubusercontent.com/111711638/186783552-321428d0-11d0-471d-94b0-30c18b6532d0.png)

![image](https://user-images.githubusercontent.com/111711638/186783862-cbfaa132-62b0-4ca7-81d0-b042f7085b8a.png)

Also tried on a second account and same thing. At this point I might as well wait till farbstoff updates rslhelper again since it seems the other developers working on this program don't see anything wrong with the current version and i've seen no advice or help other than things that i've already done mentioned above.

**Comment by Dwza** [comment]
> Well deleted rslhelper along with the folder that has the data
there should be 2 folders. also you should try to restart your pc with cleaning your memory...
in taskmanager you can see if it was resettet or not.
mostly it isnt (when fast restart is active).

**Comment by 3bee88** [comment]
Taking your latest advice of deleting everything with rslhelper in its name, disk reformatted using cc cleaner, restarted pc and then re-downloading rslhelper seems to have fixed the issue. Not sure what the real cause of the problem was since what you adviced felt like a last resort thing to do. I also noticed before that my build of rslhelper was latest version build 8, but when i did all the restarting it became latest version build 7. Maybe build 8 has some issues. I'm not gonna risk doing the update back to 8 if that is the issue.

For kojikeneda if you do the same thing as I just did maybe it will help you fix your issue with rslhelper as well.
